From 8851582ee320a10cc8cab72664dac1e139485298 Mon Sep 17 00:00:00 2001
From: Phil Sutter <psutter@redhat.com>
Date: Fri, 17 Jul 2026 11:14:18 +0200
Subject: [PATCH] tests: py: Drop stale entries since redundant test case
 removal

JIRA: https://issues.redhat.com/browse/RHEL-190549
Upstream Status: nftables commit 7db422da28349edcdeb3c96a410b242856ca74ac

commit 7db422da28349edcdeb3c96a410b242856ca74ac
Author: Phil Sutter <phil@nwl.cc>
Date:   Wed Aug 13 14:14:45 2025 +0200

    tests: py: Drop stale entries since redundant test case removal

    Fixed commit left stale JSON equivalents and payload records in place,
    drop them.

    Fixes: ec1ea13314fa5 ("tests: remove redundant test cases")
    Signed-off-by: Phil Sutter <phil@nwl.cc>
    Reviewed-by: Pablo Neira Ayuso <pablo@netfilter.org>

Signed-off-by: Phil Sutter <psutter@redhat.com>
---
 tests/py/any/meta.t.json        |  38 ----------
 tests/py/ip6/dst.t.json         |  23 ------
 tests/py/ip6/dst.t.payload.inet |  10 ---
 tests/py/ip6/dst.t.payload.ip6  |   8 ---
 tests/py/ip6/frag.t.json        | 121 --------------------------------
 tests/py/ip6/ip6.t.json         |  40 -----------
 6 files changed, 240 deletions(-)

diff --git a/tests/py/any/meta.t.json b/tests/py/any/meta.t.json
index d50272d..b315015 100644
--- a/tests/py/any/meta.t.json
+++ b/tests/py/any/meta.t.json
@@ -2294,44 +2294,6 @@
     }
 ]
 
-# meta cgroup {1048577-1048578}
-[
-    {
-        "match": {
-            "left": {
-                "meta": { "key": "cgroup" }
-            },
-            "op": "==",
-            "right": {
-                "set": [
-                    {
-                        "range": [ 1048577, 1048578 ]
-                    }
-                ]
-            }
-        }
-    }
-]
-
-# meta cgroup != { 1048577-1048578}
-[
-    {
-        "match": {
-            "left": {
-                "meta": { "key": "cgroup" }
-            },
-            "op": "!=",
-            "right": {
-                "set": [
-                    {
-                        "range": [ 1048577, 1048578 ]
-                    }
-                ]
-            }
-        }
-    }
-]
-
 # meta iif . meta oif { "lo" . "lo" }
 [
     {
diff --git a/tests/py/ip6/dst.t.json b/tests/py/ip6/dst.t.json
index e947a76..7ba4572 100644
--- a/tests/py/ip6/dst.t.json
+++ b/tests/py/ip6/dst.t.json
@@ -290,26 +290,3 @@
         }
     }
 ]
-
-# dst hdrlength != { 33, 55, 67, 88}
-[
-    {
-        "match": {
-            "left": {
-                "exthdr": {
-                    "field": "hdrlength",
-                    "name": "dst"
-                }
-            },
-            "op": "!=",
-            "right": {
-                "set": [
-                    33,
-                    55,
-                    67,
-                    88
-                ]
-            }
-        }
-    }
-]
diff --git a/tests/py/ip6/dst.t.payload.inet b/tests/py/ip6/dst.t.payload.inet
index 90d6bda..b2a79ed 100644
--- a/tests/py/ip6/dst.t.payload.inet
+++ b/tests/py/ip6/dst.t.payload.inet
@@ -119,13 +119,3 @@ ip6 test-ip6 input
   [ cmp eq reg 1 0x0000000a ]
   [ exthdr load ipv6 1b @ 60 + 1 => reg 1 ]
   [ lookup reg 1 set __set%d ]
-
-# dst hdrlength != { 33, 55, 67, 88}
-__set%d test-ip6 3
-__set%d test-ip6 0
-	element 00000021  : 0 [end]	element 00000037  : 0 [end]	element 00000043  : 0 [end]	element 00000058  : 0 [end]
-ip6 test-ip6 input
-  [ meta load nfproto => reg 1 ]
-  [ cmp eq reg 1 0x0000000a ]
-  [ exthdr load ipv6 1b @ 60 + 1 => reg 1 ]
-  [ lookup reg 1 set __set%d 0x1 ]
diff --git a/tests/py/ip6/dst.t.payload.ip6 b/tests/py/ip6/dst.t.payload.ip6
index 941140d..5058b46 100644
--- a/tests/py/ip6/dst.t.payload.ip6
+++ b/tests/py/ip6/dst.t.payload.ip6
@@ -89,11 +89,3 @@ __set%d test-ip6 0
 ip6 test-ip6 input
   [ exthdr load ipv6 1b @ 60 + 1 => reg 1 ]
   [ lookup reg 1 set __set%d ]
-
-# dst hdrlength != { 33, 55, 67, 88}
-__set%d test-ip6 3
-__set%d test-ip6 0
-	element 00000021  : 0 [end]	element 00000037  : 0 [end]	element 00000043  : 0 [end]	element 00000058  : 0 [end]
-ip6 test-ip6 input
-  [ exthdr load ipv6 1b @ 60 + 1 => reg 1 ]
-  [ lookup reg 1 set __set%d 0x1 ]
diff --git a/tests/py/ip6/frag.t.json b/tests/py/ip6/frag.t.json
index b8c06df..6953e87 100644
--- a/tests/py/ip6/frag.t.json
+++ b/tests/py/ip6/frag.t.json
@@ -230,46 +230,6 @@
     }
 ]
 
-# frag reserved { 33-55}
-[
-    {
-        "match": {
-            "left": {
-                "exthdr": {
-                    "field": "reserved",
-                    "name": "frag"
-                }
-            },
-	    "op": "==",
-            "right": {
-                "set": [
-                    { "range": [ 33, 55 ] }
-                ]
-            }
-        }
-    }
-]
-
-# frag reserved != { 33-55}
-[
-    {
-        "match": {
-            "left": {
-                "exthdr": {
-                    "field": "reserved",
-                    "name": "frag"
-                }
-            },
-            "op": "!=",
-            "right": {
-                "set": [
-                    { "range": [ 33, 55 ] }
-                ]
-            }
-        }
-    }
-]
-
 # frag frag-off 22
 [
     {
@@ -384,46 +344,6 @@
     }
 ]
 
-# frag frag-off { 33-55}
-[
-    {
-        "match": {
-            "left": {
-                "exthdr": {
-                    "field": "frag-off",
-                    "name": "frag"
-                }
-            },
-	    "op": "==",
-            "right": {
-                "set": [
-                    { "range": [ 33, 55 ] }
-                ]
-            }
-        }
-    }
-]
-
-# frag frag-off != { 33-55}
-[
-    {
-        "match": {
-            "left": {
-                "exthdr": {
-                    "field": "frag-off",
-                    "name": "frag"
-                }
-            },
-            "op": "!=",
-            "right": {
-                "set": [
-                    { "range": [ 33, 55 ] }
-                ]
-            }
-        }
-    }
-]
-
 # frag reserved2 1
 [
     {
@@ -601,44 +521,3 @@
         }
     }
 ]
-
-# frag id { 33-55}
-[
-    {
-        "match": {
-            "left": {
-                "exthdr": {
-                    "field": "id",
-                    "name": "frag"
-                }
-            },
-	    "op": "==",
-            "right": {
-                "set": [
-                    { "range": [ 33, 55 ] }
-                ]
-            }
-        }
-    }
-]
-
-# frag id != { 33-55}
-[
-    {
-        "match": {
-            "left": {
-                "exthdr": {
-                    "field": "id",
-                    "name": "frag"
-                }
-            },
-            "op": "!=",
-            "right": {
-                "set": [
-                    { "range": [ 33, 55 ] }
-                ]
-            }
-        }
-    }
-]
-
diff --git a/tests/py/ip6/ip6.t.json b/tests/py/ip6/ip6.t.json
index 49e5a2d..72d91cc 100644
--- a/tests/py/ip6/ip6.t.json
+++ b/tests/py/ip6/ip6.t.json
@@ -571,46 +571,6 @@
     }
 ]
 
-# ip6 nexthdr { 33-44}
-[
-    {
-        "match": {
-            "left": {
-                "payload": {
-                    "field": "nexthdr",
-                    "protocol": "ip6"
-                }
-            },
-	    "op": "==",
-            "right": {
-                "set": [
-                    { "range": [ 33, 44 ] }
-                ]
-            }
-        }
-    }
-]
-
-# ip6 nexthdr != { 33-44}
-[
-    {
-        "match": {
-            "left": {
-                "payload": {
-                    "field": "nexthdr",
-                    "protocol": "ip6"
-                }
-            },
-            "op": "!=",
-            "right": {
-                "set": [
-                    { "range": [ 33, 44 ] }
-                ]
-            }
-        }
-    }
-]
-
 # ip6 nexthdr 33-44
 [
     {
